lib/Doctrine/Migrations/Tools/Console/Command/DiffCommand.php

```diff
@@ -114,6 +115,16 @@ public function execute(
 
         assert(is_string($namespace));
 
+        $statusCalculator              = $this->getDependencyFactory()->getMigrationStatusCalculator();
+        $executedUnavailableMigrations = $statusCalculator->getExecutedUnavailableMigrations();
+        $newMigrations                 = $statusCalculator->getNewMigrations();
+
+        if ($this->checkNewMigrationsOrExecutedUnavailable($newMigrations, $executedUnavailableMigrations, $input, $output) === false) {
```

Comment:
That method returns a boolean, so does the `!` operator, as surely as comparing with another boolean with `===` does.
```suggestion
        if (!$this->checkNewMigrationsOrExecutedUnavailable($newMigrations, $executedUnavailableMigrations, $input, $output)) {
```